src/main/java/mate/academy/config/SecurityConfig.java

```diff
@@ -22,4 +30,22 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
         auth.userDetailsService(userDetailsService)
                 .passwordEncoder(passwordEncoder);
     }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .httpBasic().disable()
+                .csrf().disable()
+                .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .authorizeRequests()
+                .antMatchers("/registration", "/login", "/inject").permitAll()
```

Comment:
```suggestion
                .antMatchers(HttpMethod.GET, "/products", "/users").hasRole(ROLE_USER)
                .antMatchers(HttpMethod.POST, "/products").hasRole(ROLE_USER)
```
add constants for role names